src/libraries/Microsoft.Extensions.DependencyInjection.Abstractions/src/ServiceDescriptor.cs

```diff
@@ -152,24 +152,20 @@ private ServiceDescriptor(Type serviceType, object? serviceKey, ServiceLifetime
         private Type? _implementationType;
 
         /// <summary>
-        /// Gets the <see cref="Type"/> that implements the service.
+        /// Gets the <see cref="Type"/> that implements the service,
+        /// or returns <see langword="null"/> if <see cref="IsKeyedService"/> is <see langword="true"/>.
         /// </summary>
+        /// If <see cref="IsKeyedService"/> is <see langword="true"/>, <see cref="KeyedImplementationType"/> should be called instead.
```

Comment:
NIT: Might wanted to put inside `<remarks>` tag? Here and all similar cases below
```suggestion
        /// <remarks>
        /// If <see cref="IsKeyedService"/> is <see langword="true"/>, <see cref="KeyedImplementationType"/> should be called instead.
        /// <remarks/>
```

Reply:
Thanks. Done